omc3/model/model_creators/lhc_model_creator.py

```diff
@@ -28,24 +30,64 @@
     TWISS_DAT,
     TWISS_ELEMENTS_BEST_KNOWLEDGE_DAT,
     TWISS_ELEMENTS_DAT,
-    ACCELERATOR_MODEL_REPOSITORY
+    PATHFETCHER, AFSFETCHER, GITFETCHER, LSAFETCHER,
+    ACCELERATOR_MODEL_REPOSITORY,
+    MODIFIER_BRANCH,
 )
-from omc3.model.model_creators.abstract_model_creator import ModelCreator
+from omc3.model.model_creators.abstract_model_creator import ModelCreator, check_folder_choices
 from omc3.utils import iotools
 
-LHC_REPOSITORY_NAME = "acc-models-lhc"
+B2_ERRORS_ROOT = pathlib.Path("/afs/cern.ch/eng/sl/lintrack/error_tables/")
 
 LOGGER = logging.getLogger(__name__)
 
 
 def _b2_columns() -> List[str]:
-    cols_outer = [f"{KP}{num}{S}L" for KP in ("K", "P") for num in range(21) for S in ("", "S")]
+    cols_outer = [f"{KP}{num}{S}L" for KP in (
+        "K", "P") for num in range(21) for S in ("", "S")]
     cols_middle = ["DX", "DY", "DS", "DPHI", "DTHETA", "DPSI", "MREX", "MREY", "MREDX", "MREDY",
                    "AREX", "AREY", "MSCALX", "MSCALY", "RFM_FREQ", "RFM_HARMON", "RFM_LAG"]
     return cols_outer[:42] + cols_middle + cols_outer[42:]
 
 
+def is_folder(path: Path) -> bool:
+    return path.is_dir()
+
+
 class LhcModelCreator(ModelCreator):
+    acc_model_name = "lhc"
+
+    @classmethod
+    def get_options(cls, accel_inst, opt) -> bool:
```

Comment:
It is not clear to me what options are gotten here, and what the return value means

Reply:
I'll try my best to clarify this